src/main/scala/org/hammerlab/guacamole/commands/GermlineAssemblyCaller.scala

```diff
@@ -79,14 +79,13 @@ object GermlineAssemblyCaller {
      * @param reads  Set of reads to extract sequence from
      * @param startLocus Start (inclusive) locus on the reference
      * @param endLocus End (exclusive) locus on the reference
-     * @param kmerSize Length of the subsequence
      * @param minOccurrence Mininum number of times a subsequence needs to appear to be included
      * @return List of subsequences overlapping [startLocus, endLocus) that appear at least `minOccurrence` time
      */
     def getConsensusKmer(reads: Seq[MappedRead],
                          startLocus: Int,
                          endLocus: Int,
-                         minOccurrence: Int): Iterable[Array[Byte]] = {
+                         minOccurrence: Int): Iterable[ContigSequence] = {
```

Comment:
I'm not sure it makes sense to change `Array[Byte]` types to `ContigSequence` unless they are truly the sequence of a contig, i.e. they are indexed by genomic locus. Looking at this it would appear to me this function is returning an entire contig's sequence